.github/workflows/test.yml

```diff
@@ -40,3 +37,16 @@ jobs:
       - name: Test Format
         run: cargo fmt -- --check
       - run: cargo clippy --features unstable --tests -- -Dclippy::all
+  windows:
```

Comment:
forgot to mention, but `near-sdk-macros` also compiles with windows. It's just sim that doesn't

Reply:
yeah, that's fine. It all gets checked and compiled through `near-sdk` anyway